spec/features/users/03_user_signs_out_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+feature 'user signs in', %{
+  As a user
+  I want to sign out
+  So that I can securely leave the site
+} do
+
+  # Acceptance Criteria:
+  # - [ ] if I am logged in, I can log out
+  # - [ ] if I am not logged in, there is no option to log out
+  # - [ ] if I sign out, I can sign back in
+
+  let (:user) { FactoryGirl.create(:user) }
+
+  scenario 'a signed-in user logs out' do
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log In'
```

Comment:
I think it would make sense to leave a blank line between Log In and Sign Out, since it's a separate page. That's nitpicky, but I like having blank lines to denote page loads in my feature tests.